### PR TITLE
POC

### PR DESCRIPTION
**Pull request by JOduMonT**
I make it few time on Debian 8 and 9, Centos 7 and Ubuntu 16.04LTS

### CODE REVIEW

File: INSTALL_DEBIAN.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+## HOWTO
+Where basicly reproduce what the user_data do for us.
+
+## Let's make a step-by-step humanable readable.
+as root
+
+### SSHD Config
+Don't forget to create the user core and adding your ssh key before
+You could also remove AllowUsers core or/and change the username.
+> echo "UsePrivilegeSeparation sandbox
```
Comment (pierreozoux) [inline INSTALL_DEBIAN.md]: Could you use

```bash
write your code here
```

I find it more readable?

Also, I think an heredoc would be better here:

cat > myfile <<EOF
line1
line2
EOF

**Comment by pierreozoux** [comment]
You could use coreos-cloudinit also in debian, just a matter to install the bin :)

But yeah, thanks a lot, it is a good idea, we are actually installing it in ubuntu also, we didn't document it yet, but good work :)

**Comment by pierreozoux** [comment]
Just for the code syntax, can you use this way:
https://github.com/indiehosters/libre.sh/blame/master/README.md#L65-L71

I find it more readable, and this would be consistent with the rest.

Thanks!

**Comment by JOduMonT** [comment]
I'm sorry I don't get this part.
I look [INSTALL](https://github.com/indiehosters/libre.sh/blob/master/INSTALL.md) and [INSTALL_HETZNER](https://github.com/indiehosters/libre.sh/blob/master/INSTALL_HETZNER.md)
and they don't seems following what you ask, so I don't really understand why I should follow the README nomenclature inside a INSTALL HowTo ..

**Comment by pierreozoux** [comment]
In the install it is the same:
https://raw.githubusercontent.com/indiehosters/libre.sh/master/INSTALL_HETZNER.md
https://raw.githubusercontent.com/indiehosters/libre.sh/master/INSTALL.md

If you don't want, I'll do it. It is just to be consistent over code syntax.

**Comment by pierreozoux** [comment]
It doesn't, but I'll do it :)

**Pull request merged**